datarepo-clienttests/src/main/resources/suites/PRSmokeTests.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "PRSmokeTests",
-  "description": "Two smoke tests to run on PR build. One requires authentication and one does not. Only queries, no data creation.",
+  "description": "Three smoke tests to run on PR build. Two requires authentication and one does not.",
```

Comment:
Nit: I think we should call this PRNightlyTests so as not to confuse the "smoke" test suite.

Reply:
I think this is the smoke test PR run. I didn't see your comments on my last PR until too late, so I thought I'd add it here. Sorry for the confusion!

Reply:
Got it, thanks!